test/unit/upload-stream-unit.js

```diff
@@ -3,7 +3,7 @@ var expect = chai.expect;
 var UploadStream = require("../../lib/upload-stream");
 var Uploader = require("../../lib/uploader");
 var S3 = require("mock-s3").S3;
-var Writable = require("stream").Writable;
+var Writable = require("readable-stream").Writable;
```

Comment:
we can also just get rid of the test that checks inheritance of the stream object if you want